### PR TITLE
Update conditional-access-integrate-jamf.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the document as per the new Jamf pro navigation changes

Reference -https://docs.jamf.com/technical-papers/jamf-pro/microsoft-intune/10.34.0/Manually_Configure_the_Connection_Between_Jamf_Pro_and_Microsoft_Intune.html

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger5** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2377

File: memdocs/intune/protect/conditional-access-integrate-jamf.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ You need the following to configure Conditional Access with Jamf Pro:
 
 - Jamf Pro 10.1.0 or later
 - Microsoft Intune and Microsoft AAD Premium P1 licenses (recommended Microsoft Enterprise Mobility + Security license bundle)
+- Global Administrator role in Azure Active Directory.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/conditional-access-integrate-jamf.md:52]:
```suggestion
- Global admin role in Azure Active Directory.
```

File: memdocs/intune/protect/conditional-access-integrate-jamf.md [hunk 2]
```diff
@@ -107,12 +108,19 @@ To connect Intune with Jamf Pro:
 8. Next, we will add permissions to update device attributes. At the top left of the **API permissions** page, select **Add a permission** to add a new permission. 
 
 9. On the **Request API permissions** page, select **Intune**, and then select **Application permissions**. Select only the check box for **update_device_attributes** and save the new permission.
+10. Under **Microsoft Graph**, click **Application permissions**, and then select **Application.Read.All**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/conditional-access-integrate-jamf.md:111]:
```suggestion
10. Under **Microsoft Graph**, select **Application permissions**, then select **Application.Read.All**.
```

File: memdocs/intune/protect/conditional-access-integrate-jamf.md [hunk 3]
```diff
@@ -107,12 +108,19 @@ To connect Intune with Jamf Pro:
...
 
-10. Next, grant admin consent for this app by selecting **Grant admin consent for _\<your tenant>_** in the top left of the **API permissions** page. You may need to re-authenticate your account in the new window and grant the application access by following the prompts.  
+11. Click **Add permissions**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/conditional-access-integrate-jamf.md:113]:
```suggestion
11. Select **Add permissions**.
```

File: memdocs/intune/protect/conditional-access-integrate-jamf.md [hunk 4]
```diff
@@ -107,12 +108,19 @@ To connect Intune with Jamf Pro:
...
 
-11. Refresh the page by click on the **Refresh** button at the top of the page. Confirm that admin consent has been granted for the **update_device_attributes** permission. 
+12. Navigate to APIs my organization uses. Search for and click **Windows Azure Active Directory**. Click **Application permissions**, and then select **Application.Read.All**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/conditional-access-integrate-jamf.md:115]:
```suggestion
12. Navigate to **APIs my organization uses**. Search for and select **Windows Azure Active Directory**. Select **Application permissions**, and then select **Application.Read.All**.
```

File: memdocs/intune/protect/conditional-access-integrate-jamf.md [hunk 5]
```diff
@@ -107,12 +108,19 @@ To connect Intune with Jamf Pro:
...
 
-12. After the app is registered successfully, the API permissions should only contain one permission called **update_device_attributes** and should appear as follows:
+13. Click **Add permissions**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/conditional-access-integrate-jamf.md:117]:
```suggestion
13. Select **Add permissions**.
```

File: memdocs/intune/protect/conditional-access-integrate-jamf.md [hunk 6]
```diff
@@ -107,12 +108,19 @@ To connect Intune with Jamf Pro:
...
+
+14. Next, grant admin consent for this app by selecting **Grant admin consent for _\<your tenant>_** in the top left of the **API permissions** page. You may need to re-authenticate your account in the new window and grant the application access by following the prompts.  
+
+15. Refresh the page by click on the **Refresh** button at the top of the page. Confirm that admin consent has been granted for the **update_device_attributes** permission.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/conditional-access-integrate-jamf.md:121]:
```suggestion
15. Refresh the page by selecting **Refresh** at the top of the page. Confirm that admin consent has been granted for the **update_device_attributes** permission. 
```

File: memdocs/intune/protect/conditional-access-integrate-jamf.md [hunk 7]
```diff
@@ -136,9 +144,13 @@ The app registration process in Azure AD is complete.
 1. Activate the connection in the Jamf Pro console:
 
    1. Open the Jamf Pro console and navigate to **Global Management** > **Conditional Access**. Click the **Edit** button on the **macOS Intune Integration** tab.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/conditional-access-integrate-jamf.md:146]:
```suggestion
   1. Open the Jamf Pro console and navigate to **Global Management** > **Conditional Access**. Select **Edit** on the **macOS Intune Integration** tab.
```

File: memdocs/intune/protect/conditional-access-integrate-jamf.md [hunk 8]
```diff
@@ -136,9 +144,13 @@ The app registration process in Azure AD is complete.
...
-   2. Select the check box for **Enable Intune Integration for macOS**.
-   3. Provide the required information about your Azure tenant, including **Location**, **Domain name**, the **Application ID**, and the value for the *client secret* that you saved when you created the app in Azure AD.
-   4. Select **Save**. Jamf Pro tests your settings and verifies your success.
+   2. Select the check box for **Enable Intune Integration for macOS**. When this setting is enabled, Jamf Pro sends inventory updates to Microsoft Intune. Clear the selection if you want to disable the connection but save your configuration.
+   3. Select **Manua** under Connection Type.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/conditional-access-integrate-jamf.md:148]:
```suggestion
   3. Select **Manual** under **Connection type**.
```

File: memdocs/intune/protect/conditional-access-integrate-jamf.md [hunk 9]
```diff
@@ -136,9 +144,13 @@ The app registration process in Azure AD is complete.
...
+   4. From the **Sovereign Cloud** pop-up menu, select the location of your Sovereign Cloud from Microsoft.
+   5. Click **Open administrator consent URL**, and follow the onscreen instructions to allow the Jamf Native macOS Connector app to be added to your Azure AD tenant.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/conditional-access-integrate-jamf.md:150]:
```suggestion
   5. Select **Open administrator consent URL** and follow the onscreen instructions to allow the Jamf Native macOS Connector app to be added to your Azure AD tenant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger6** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger-2** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**